docs/softlayer_virtual_guest.md

```diff
@@ -78,8 +78,8 @@ The following arguments are supported:
 * `os_reference_code` | *string*
     * An operating system reference code that will be used to provision the computing instance.
     * **Conflicts with** `image_id`.
-* `image_id` | *string*
-    * A global identifier for the image template to be used to provision the computing instance.
+* `image_id` | *int*
+    * The image template id to be used to provision the computing instance. Note this is not the global identifier (uuid), but the image template group id that should point to a valid global identifier.
     * **Conflicts with** `os_reference_code`.
```

Comment:
FYI - in a parallel branch for the new ssh key datasource, I added a `**Note:**` immediately following the `ssh_keys` attribute description with information about how to obtain the key IDs (in my case, by referencing the data source documentation page).  If you prefer another style, let me know so I can adjust, so that within the one doc page, there is consistency.  https://github.com/softlayer/terraform-provider-softlayer/blob/feature/datasource_ssh_keys/docs/softlayer_virtual_guest.md  (by the way, the link in my note won't work until the code is merged to master)